gui/default/syncthing/core/syncthingController.js

```diff
@@ -1402,6 +1402,11 @@ angular.module('syncthing.core')
                 autoNormalize: true
             };
             $scope.editingExisting = false;
+            $scope.newIgnores = {
+                edited: false,
```

Comment:
Also, I feel the end state would either be load stuff for unadded folders and allow modifying (but that's a security hole), or overwriting